src/dmd/expressionsem.d

```diff
@@ -9510,7 +9510,7 @@ Expression semanticX(DotIdExp exp, Scope* sc)
 
 // Resolve e1.ident without seeing UFCS.
 // If flag == 1, stop "not a property" error and return NULL.
-Expression semanticY(DotIdExp exp, Scope* sc, int flag)
+Expression semanticY(DotIdExp exp, Scope* sc, int flag, bool calledForAliasThis = false)
```

Comment:
Walter won’t like this optional argument.

Reply:
I haven't found a better way to signal that the function was called from resolveAliasThis